utilities/hasRichText.ts

```diff
@@ -1,11 +1,10 @@
-import { render } from 'storyblok-rich-text-react-renderer-ts';
-import { getNumBloks } from './getNumBloks';
+import { render, StoryblokRichtext } from 'storyblok-rich-text-react-renderer-ts';
 
 /**
  * Returns a boolean if the Storyblok rich text field (wysiwyg) has any content.
  *
- * @param {object} wysiwyg - The wysiwyg to check for content.
+ * @param {StoryblokRichtext} wysiwyg - The wysiwyg to check for content.
  * @returns {boolean} - Whether or not the wysiwyg has content.
  */
 
-export const hasRichText = (wysiwyg) => getNumBloks(render(wysiwyg)) > 0;
+export const hasRichText = (wysiwyg: StoryblokRichtext) => !!render(wysiwyg);
```

Comment:
There is a `StoryblokRichtext` type provided by the renderer. Also revised the hasRichText function since we don't actually need the `getNumBloks` function for the check